x/participationrewards/keeper/keeper_test.go

```
@@ -35,10 +35,19 @@
 	umeeTestChain      = "umee-types-1"
 	umeeBaseDenom      = "uumee"
 
+	osmosisTestChain = "osmosis-1"
+	osmosisBaseDenom = "uosmo"
```

golangci-lint (1.62.2)

var `osmosisBaseDenom` is unused

(unused)